infra/resources/modules/commons/api/ts_management/v1/_openapi.yaml

```diff
@@ -439,6 +500,36 @@ components:
       enum:
         - CREATING
         - CREATED
+    NextId:
+      type: string
+      description: Unique identifier of the next item.
+      minLength: 1
+    PreviousId:
```

Comment:
can we define id with a more strict typings? i.e: 
```
maxLength: 26
minLength: 26
```

Reply:
since we now have 3 schemas (trialId, nextId and previousId) with the same type, would it be ok to create a schema (ItemId) from which the formers will inherit the typing?
There's a comment in row 296 (Using the reference ($ref) to UserId breaks the client generator) but it seems that using $ref in a schema doesn't break `yarn generate`.

Reply:
yes it can be used as a common schema

Reply:
resolved in 2b5330db8f4707a14c57fc0cd171946bc6a7988d and 35d9506f1e1ad11c925b87f4b10e7501f4236571 (stricter typing in TrialId breaks the build)